libgstc/python/meson.build

```diff
@@ -1 +1,25 @@
-run_command('pip3', 'install', '.')
+pymod = import('python')
+python = pymod.find_installation(
+  get_option('with-python-version'),
+  required : get_option('enable-python').enabled(),
```

Comment:
This accepts a feature option directly, no need to convert it into a boolean.

And properly using feature options means that when passing `-Denable-python=disabled`, this python installation will be skipped rather than "looked for, but not required".

... Also the "enable-" in this option name seems a bit redundant given it needs to be defined as one of:
- `-Denable-python=disabled`
- `-Denable-python=enabled`
- `-Denable-python=auto`

and the first two are... pretty confusing.

Reply:
Oh hmm, okay, I see the existing feature options use that "enable-" stuff already... never mind I guess.

:(

Reply:
> This accepts a feature option directly, no need to convert it into a boolean.

Fixed.

> Oh hmm, okay, I see the existing feature options use that "enable-" stuff already... never mind I guess.

Yeah, I had used that style for consistency.

Reply:
I agree, it is redundant. I assume this came from the autotools port. I may change them in a future branch.